Cargo.toml

```diff
@@ -39,18 +39,13 @@ eth-types = { path = "eth-types" }
 contracts = { path = "contracts" }
 preprocessor = { path = "preprocessor" }
 
-
-# halo2
-halo2-base = { git = "https://github.com/axiom-crypto/halo2-lib", tag = "v0.4.1", default-features = false, features = [
-    "halo2-pse",
```

Comment:
see, you removed features; the other two, I don't mind - but re. "halo2-pse," I guess it's the default in axiom-crypto/halo2-lib?

Reply:
My bad, I kept working on the branch after submitting the PR, will resolve that.